cbmc_viewer/symbol_table.py

```diff
@@ -13,8 +13,17 @@
 def symbol_table(goto):
     """Extract symbol table from goto binary as lines of text."""
 
+    # The --show-symbol-table flag produces a sequence of symbol
+    # definitions.  Definitions are separated by blank lines.  Each
+    # definition is a sequence lines including
+    #
+    #   Symbol......: symbol
+    #   Pretty name.: symbol
```

Comment:
Symbol and pretty name will always be the same?

Reply:
No, updated the comment.